lib/scan/slack_poster.rb

```diff
@@ -9,6 +9,11 @@ def run(results)
       notifier = Slack::Notifier.new(Scan.config[:slack_url])
       notifier.username = 'fastlane'
 
+      if options[:slack_channel].to_s.length > 0
```

Comment:
This should have been `Scan.config` instead of `options`, I fixed it on `master`